registry/server/routes/config.ts

```diff
@@ -25,7 +26,7 @@ router.get('/', async (req, res) => {
         routes: [] as any[],
         specialRoutes: {},
         settings: {},
-        routerDomains: {},
+        routerDomains,
```

Comment:
Hmm, what if instead of adding `routerDomains` property to config - we would turn `domainId: int` field into `domain: string | null` for respective routes?

Reply:
on the one hand, I think it's good idea, but on the other hand it will be a little bit harder to detect inside ILC - "do we respect the current domain".

Reply:
Hmm, why so?

Reply:
done